jboss-kie-wildfly-common/added/launch/jboss-kie-wildfly-security-login-modules.sh

```diff
@@ -114,15 +109,7 @@ function configure_role_mapper_login_module() {
     add_login_module "${login_module}"
 }
 
-function configure_keycloak_login_module() {
```

Comment:
why removing it?

Reply:
It was added only for the external_auth property. The login module is configured already by the keycloak scripts